src/txmempool.cpp

```diff
@@ -59,6 +59,8 @@ CTxMemPoolEntry::GetPriority(unsigned int currentHeight) const
 CTxMemPool::CTxMemPool(const CFeeRate& _minRelayFee) :
     nTransactionsUpdated(0)
 {
+    _clear(); //lock free clear
```

Comment:
```suggestion
    _clear(); // unlocked clear
```
"Lock-free" is incorrect. That would imply use of lock-free programming which is not the case: "lock-free" typically implies use of memory accesses relying on a low-level memory concurrency model, and not literally just "free of locking". The point of the change is that this is called in a single-threaded context where no synchronization –by locking or otherwise– is needed.